opendse-model/src/main/java/net/sf/opendse/model/TGFFSpecification.java

```diff
@@ -0,0 +1,151 @@
+package net.sf.opendse.model;
+
+import java.util.Set;
+
+import net.sf.opendse.model.parameter.Parameter;
+
+/**
+ * The {@code TGFFSpecification} consists of an {@link Application},
+ * {@link ResourceTypes}, and {@link Mappings}.
+ * 
+ * @author Valentina Richthammer
+ * 
+ */
+public class TGFFSpecification implements IAttributes {
```

Comment:
Ok. Nice. Now, we could think about interfaces for both, the normal as well as to the TGFF specification, ideas tbd. see below.